Management/src/ActuatorWeb/README.md

```diff
@@ -0,0 +1,212 @@
+All of the above is also true for ActuatorApi, except that application has actuators configured on port 8091.
+
+### Zipkin Server
+
+> [!NOTE]  
+> Previous versions of Steeltoe included distributed tracing functionality. That functionality has moved to OpenTelemetry.
+> We expect many Steeltoe users still use distributed tracing, so this sample includes a basic OpenTelemetry setup.
+> Visit [OpenTelemetry](https://opentelemetry.io/docs/languages/net/) to learn more.
+
+These applications use [OpenTelemetry extension methods](./OpenTelemetryExtensions.cs) to instrument all HTTP interactions with
+trace information and export those traces to Zipkin so that individual requests across the system can be analyzed. Refer
+to [Common Tasks](../../../CommonTasks.md#zipkin) to start Zipkin.
+
+### Starting the application
+
+You should now be able to run this sample, either from your IDE or with this command:
+
+```shell
+dotnet run
+```
+
+> [!NOTE]
+> If you are only looking for an actuator demo, you _may_ proceed from here without starting ActuatorApi, but for full
+> functionality, review the [ActuatorApi README](../ActuatorApi/README.md) and get that app running as well.
+
+### Interacting with the application
+
+As ActuatorWeb is a Razor Pages app, you can use a browser to [access the web UI](http://localhost:5126) while the app
+is running. To request a weather forecast (and generate HTTP request traces) from ActuatorApi, click "Weather Forecast"
+in the site menu.
+
+> [!NOTE]
+> The Weather Forecast API endpoint is delayed randomly up to three seconds to generate interesting trace data.
+
+### Interacting with Management Endpoints
+
+Each app instance registers with Spring Boot Admin and can be viewed from the web interface at <http://localhost:9099>.
+
+[ActuatorWeb.http](./ActuatorWeb.http) and [ActuatorApi.http](../ActuatorApi/ActuatorApi.http) are also provided for
+experimentation with the exposed actuators. Variables for the .http files are stored
+in [http-client.env.json](./http-client.env.json). In both Visual Studio and Rider, when you open the .http file there
+is a dropdown box that can be used to select the environment you're working in (for example: running locally with actuators listening on an alternate port with basic authentication on). The dropdown is probably at the top of the window and may
+initially look like `env: <none>` or `Run With: No Environment`, although those interfaces are subject to change over
+time. You will need to either make an environment selection or define variables like `HostAddress` directly in the .http
+file in order for requests to work (review the .json file mentioned previously for variable examples). Please note that
+the "dhaka" environment is not expected to be available for use by those not on the Steeltoe team.
+
+#### Regarding HTTPS and Basic Authentication
```

Comment:
Notes are intended for extra explanations in flowing text, not the entire section should be a note.
Use `[!WARNING]` or `[!CAUTION]` to draw attention, instead of making text uppercase or bold.
The "something else" is rather vague, consider "a different authorization mechanism".
A link named "Basic Authentication" should point to what basic auth means, not the library we're using.

Reply:
All except using alerts were addressed. The bold text is barely distinguishable in dark mode.